articles/quickstart/spa/auth0-react/01.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to a React application using Auth0.
```

Comment:
What if I'm using typescript? Will that be a separate quickstart?

Reply:
👍 Mentioned that here. I think it should be a separate quickstart.

https://github.com/auth0/docs/pull/9077/files#r431108321

Reply:
See my comment above - I think we should just have one quickstart but default to TypeScript. The samples in the doc likely won't change too much, but the _sample_ will be in TypeScript for reference.

Reply:
Moved to CF.